flixel/addons/tile/FlxTilemapExt.hx

```diff
@@ -391,14 +398,25 @@ class FlxTilemapExt extends FlxTilemap
 				}
 				column++;
 			}
-
 			rowStart += widthInTiles;
 			row++;
 		}
 
 		return results;
 	}
-
+	/**
+	 * Set glue to force contact with slopes and a slow down factor while climbing 
+	 *
+	 * @param 	downwardsGlue  Activate/Deactivate glue on slopes on the
+	 * @param 	slopeSlowDownFactor  A slowing down factor while climbing slopes, from 0.0 to 1.0, By default 0.0, no slow down.
```

Comment:
Did you remove the default value for this parameter intentionally? Either the `@param` docs have to be updated, or the default value of `0` should be restored.